lib/spack/spack/config.py

```diff
@@ -101,6 +101,7 @@
         'checksum': True,
         'dirty': False,
         'build_jobs': min(16, multiprocessing.cpu_count()),
+        'build_stage': '$spack/var/spack/stage',
```

Comment:
FWIW, we try to keep these consistent with whatever is in `etc/spack/defaults`.  I realize it's redundant.

Reply:
Okay.  Will change to the same setting above (per offline discussion).